testgrid/specs/storage-migration.yaml

```diff
@@ -51,6 +51,10 @@
       echo Rook namespace found after the upgrade.
       exit 1
     fi
+    if [ -d "/var/lib/rook" ] || [ -d "/opt/replicated/rook" ]; then
```

Comment:
I assume we expect this to fail at this point @ricardomaraschini  @camilamacedo86 ?

https://github.com/replicatedhq/kURL/blob/61580229c2758349b8ac3efb39e22bdff19298dc/scripts/common/rook.sh#L121

Reply:
Ouch, thanks for the heads-up @emosbaugh . Seems like we never delete those directories so all tests moving the cluster from Rook to OpenEBS will fail. @camilamacedo86 when you proposed this change what did you expect ? The directories to be removed or that we will eventually patch the migration code to remove them ?

Reply:
Oh, nice. Thanks.